esp-hal-common/src/rtc_cntl/mod.rs

```diff
@@ -106,6 +99,7 @@ impl Clock for RtcSlowClock {
 }
 
 #[cfg(not(esp32c6))]
+#[allow(unused)]
```

Comment:
Why do we need `#[allow(unused)]` here? If this type is not used by a certain chip then it should be `cfg`'d away for that chip.